### PR TITLE
Update index.d.ts

### PR DESCRIPTION
**Pull request by k8w**
SetOption value should be any (maybe boolean, number ...)

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [x] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [x] Delete the package's directory.
- [x] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Review by micksatana — CHANGES_REQUESTED** [review]
Seem correct.

Should update project url as well? It changed to https://github.com/showdownjs/showdown

**Comment by RyanCavanaugh** [comment]
@k8w Please address comments from the code reviewers.

File: types/showdown/index.d.ts [hunk 1]
```diff
@@ -300,7 +300,7 @@ declare namespace Showdown {
          * @param optionKey
          * @param string
```
Comment (tan9) [inline types/showdown/index.d.ts]: Please update the jsdoc according to your change.

**Comment by k8w** [comment]
@tan9 done

**Pull request merged**